<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>한컴 말랑말랑 Bold</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Intelligent</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 85, 127, 255));</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>한컴 말랑말랑 Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Dongyang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>배달의민족 도현</family>
              </font>
             </property>
             <property name="text">
              <string>순차 작동</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Line" name="line_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="move_2">
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>행동1</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_51_2">
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Point1</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_1">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>-&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_52_2">
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Point2</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_54">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>-&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_53_2">
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Point3</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_56">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>-&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_54_4">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>나눔고딕</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Point4</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="move_1">
                 <property name="text">
                  <string>모터1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="move_6">
                 <property name="text">
                  <string>모터4</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="move_8">
                 <property name="text">
                  <string>모터2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="move_9">
                 <property name="text">
                  <string>챔버 열기/UV끄기</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QPushButton" name="move_10">
                 <property name="text">
                  <string>모터3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="move_11">
                 <property name="text">
                  <string>챔버 닫기/UV 작동</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>배달의민족 도현</family>
              </font>
             </property>
             <property name="text">
              <string>개별작동</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="Line" name="line_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>배달의민족 도현</family>
                </font>
               </property>
               <property name="text">
                <string>작동 여부</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="main_move">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>배달의민족 도현</family>
                 <pointsize>30</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame{
	background-color:rgb(241,95,95)
}</string>
               </property>
               <property name="text">
                <string>정지</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>모터1</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>모터2</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>모터3</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>모터4</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="motor_1">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	background-color:rgb(241,95,95)
}
</string>
                 </property>
                 <property name="text">
                  <string>정지</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="motor_2">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	background-color:rgb(241,95,95)
}
</string>
                 </property>
                 <property name="text">
                  <string>정지</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="motor_3">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	background-color:rgb(241,95,95)
}</string>
                 </property>
                 <property name="text">
                  <string>정지</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="motor_4">
                 <property name="font">
                  <font>
                   <family>배달의민족 도현</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	background-color:rgb(241,95,95)
}</string>
                 </property>
                 <property name="text">
                  <string>정지</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>배달의민족 도현</family>
              </font>
             </property>
             <property name="text">
              <string>개별 동작 여부</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
